src/openkit-shared/Util/IEnumerableExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿//
+// Copyright 2018 Dynatrace LLC

```

Comment:
duplicate file ?
src/openkit-shared/Util/IEnumerableExtensions.cs
src/openkit-shared/Core/Util/IEnumerableExtensions.cs

Reply:
Good finding, thanks!
Indeed it's a duplicate and `src/openkit-shared/Util/IEnumerableExtensions.cs` will remain.